scalableminds/webknossos
# Opt-in webgl antialiasing

### URL of deployed dev instance (used for testing):
- https://optionalantialiasing.webknossos.xyz

### Steps to test:
- open a dataset --> antialiasing should be disabled (check out diagonal lines in the 3d viewport)
- enable the antialiasing setting in the left sidebar
- reload the page --> antialiasing should be enabled

### Issues:
- follow-up to #6748 which disabled antialiasing due to a rendering bug for which another workaround was adapted, too. performance-wise it makes sense to make this setting opt-in in my opinion.

------
(Please delete unneeded items, merge only when none are left open)
